src/Microsoft.Health.Dicom.Core/Features/Store/StoreService.cs

```diff
@@ -267,7 +268,7 @@ private async Task DisposeResourceAsync(int index)
         }
     }
 
-    private static bool EnableDropMetadata(int? version)
+    private static bool IsApiV2Enabled(int? version)
```

Comment:
We may want to clarify this language - the feature is something like `ReturnWarning202` and the condition is it will be valid for all API versions greater than 1. Most of the naming is focused on the condition instead of the feature.